mllib/src/main/scala/org/apache/spark/mllib/classification/LogisticRegression.scala

```diff
@@ -374,4 +383,82 @@ class LogisticRegressionWithLBFGS
       new LogisticRegressionModel(weights, intercept, numFeatures, numOfLinearPredictor + 1)
     }
   }
+
+  /**
+   * Run the algorithm with the configured parameters on an input RDD
+   * of LabeledPoint entries starting from the initial weights provided.
```

Comment:
Replace `algorithm` by `Logistic Regression`, and remove `starting from the initial weights provided`.

Add a new line between `of LabeledPoint entries` and `If a known updater is used`.

Actually, in ml version, disabling feature scaling is supported now. So please call ml implementation in this case.